tests/foreman/ui/test_contentview.py

```diff
@@ -3089,56 +3071,78 @@ def test_positive_composite_child_inc_update(session, rhel7_contenthost, default
 
     :CaseLevel: Integration
     """
-    repo_name = gen_string('alphanumeric')
-    repo_url = create_repo(repo_name, settings.repos.inc_upd.url, [FAKE_0_INC_UPD_OLD_PACKAGE])
-    result = repo_add_updateinfo(
-        repo_name, f'{settings.repos.inc_upd.url}{FAKE_0_INC_UPD_OLD_UPDATEFILE}'
-    )
-    assert result.status == 0
     org = entities.Organization().create()
     lce = entities.LifecycleEnvironment(organization=org).create()
+    product = entities.Product(organization=org).create()
+    yum_repo_name = gen_string('alpha')
+    # Creates custom yum repository
+    yum_repo = entities.Repository(
+        name=yum_repo_name,
+        url=settings.repos.yum_1.url,
+        content_type=REPO_TYPE['yum'],
+        product=product,
+    ).create()
+    product.sync()
+    # creating cv, cv filter, and publish cv
+    cv = entities.ContentView(organization=org, repository=[yum_repo]).create()
+    cvf = entities.RPMContentViewFilter(
+        content_view=cv, inclusion=False, name=gen_string('alphanumeric')
+    ).create()
+    entities.ContentViewFilterRule(
+        content_view_filter=cvf,
+        name='walrus',
+        version='5.21',
+    ).create()
+    cv.publish()
+    cvv = entities.ContentView(id=cv.id).read().version[0]
+    promote(cvv, lce.id)
+    # Setup tools repo and add it to ak
     repos_collection = RepositoryCollection(
-        distro=DISTRO_RHEL7, repositories=[SatelliteToolsRepository(), YumRepository(url=repo_url)]
+        distro=constants.DISTRO_RHEL7, repositories=[SatelliteToolsRepository()]
     )
     content_data = repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
+    # adding custom repo subscription to ak
+    ak_id = content_data['activation_key']['id']
+    command = (
+        f'hammer activation-key add-subscription --id {ak_id} '
+        f'--subscription {product.name} --organization-id {org.id}'
+    )
+    result = default_sat.execute(command)
```

Comment:
You could have used nailgun/API for faster performance https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L282 

Non-blocking comment though !